test/Microsoft.ML.TestFrameworkCommon/Microsoft.ML.TestFrameworkCommon.csproj

```diff
@@ -8,4 +8,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelVersion)" />
```

Comment:
Why is this necessary?

Reply:
It was needed in the publish step due to a version conflict with the code coverage nuget we are using. I have since switched to not need to actually run the `publish` command so I will remove it.